src/site/_includes/partials/subscribe.njk

```diff
@@ -1,10 +1,10 @@
 <web-subscribe id="subscribe">
   <div class="w-subscribe">
-    <div class="w-display--flex w-subscribe--padded w-text--center">
-      <h1 class="w-headline--two">Helping you build powerful web experiences</h1>
+    <div class="w-display--flex w-text--center">
+      <h1 class="w-subscribe--margin">Developer Newsletter</h1>
```

Comment:
instead of using a unique class here, what do you think about adding a class to `_display.scss` that lets you set the flexbox alignment/justified mode.

`.w-justify-content--center`
`.w-align-items--center`

then you could put them on the parent of this element.